.github/workflows/publish-prerelease.yml

```diff
@@ -72,9 +72,17 @@ jobs:
           ./ci-scripts/package.sh -a ${{ matrix.arch }} -e ${{ matrix.ext }}
       - name: Archive artifact
         uses: actions/upload-artifact@v3
+        if: ${{ matrix.ext == 'bin' }}
+        with:
+          name: ceramic-one_${{ matrix.target }}
+          path: ceramic-one_${{ matrix.target }}.bin.tar.gz
+      - name: Archive artifact
+        uses: actions/upload-artifact@v3
+        if: ${{ matrix.ext == 'deb' }}
```

Comment:
Can possible take these lines out and just upload both files for all archs?
```
            ceramic-one_${{ matrix.target }}.bin.tar.gz
            ceramic-one_${{ matrix.target }}.tar.gz
```

Reply:
How?

Reply:
I'll try it out and let you know.